tcl/tson.tcl

```diff
@@ -1,5 +1,54 @@
 namespace eval qc {
-    namespace export tson_object json_quote tson2json tson_object_from tson2xml
+    namespace export tson* json_quote
+}
+
+
+proc qc::tson_string {value} {
+    #| Returns a TSON string.
+    return [list string $value]
+}
+
+proc qc::tson_number {value} {
+    #| Returns a TSON number.
+    if { [string tolower $value] in [list "" "null"] } {
+        return "null"
+    }
+    return [list number $value]
+}
+
+proc qc::tson_boolean {value} {
+    #| Returns a TSON boolean.
+    if { [string tolower $value] in [list "" "null"] } {
+        return "null"
+    }
+    return [list boolean [qc::cast boolean $value true false]]
```

Comment:
`qc::cast boolean` isn't supported in qcode-tcl-5 ... think it needs to be `qc::cast_boolean` (with underscore)

Reply:
Pushed it with a cast_boolean